code/modules/client/preference_setup/global/setting_datums.dm

```diff
@@ -95,6 +95,13 @@ var/list/_client_preferences_by_type
 	key = "DIGEST_NOISES"
 	enabled_description = "Noisy"
 	disabled_description = "Silent"
+	
+/datum/client_preference/belch_noises // Belching noises - pref toggle for 'em
+	description = "Burping"
+	key = "BELCH_NOISES"
+	enabled_description = "Noisy"
+	disabled_description = "Silent"
+	enabled_by_default = FALSE
```

Comment:
Should be true by default, but thats imo

Reply:
CHOMP has it off by default, and it works out fine. Y'hear a belch, decide you don't want it, so turn it off. <3

I'd rather it be off by default, let people decide if they dislike it to find the toggle to turn it off.

Reply:
Wait, correct me if I'm reading this wrong, but doesn't "Not enabled by default" mean nobody hears it unless they actively go to find pref and turn it on? Which is opposite of what you describe? Might be forgetting which way noise preferences go there, just making sure.

Reply:
It should be on by default. If people want to turn it off they can do that. If it's off by default most people won't even know to go find the toggle.

Reply:
I'm tired and misread it, yeah. My bad.
Will fix, one sec.

Reply:
Resolved.